main.js

```diff
@@ -75,6 +75,7 @@ map = (function () {
 
     var layer = Tangram.leafletLayer({
         scene: style_file,
+        global: { sdk_api_key: 'vector-tiles-VyYjZGS' },
```

Comment:
This doesn't work, the API key is still false:

<img width="435" alt="screen shot 2016-12-22 at 11 00 54" src="https://cloud.githubusercontent.com/assets/853051/21436947/eceafb5a-c835-11e6-9484-a4b6709069f8.png">